app/css/Visualization.css

```diff
@@ -0,0 +1,4 @@
+.webview {
+    width: 100vw;
+    min-height: 100vh;
```

Comment:
we'll definitely want to think about how we want to present these, but that's not a requirement for this PR, it's something we can tweak pretty easily.